packages/documentation/src/pages/getting-started/index.mdx

```diff
@@ -97,14 +108,46 @@ These instructions cover building and running VA.gov locally, including configur
    $ yarn fetch-drupal-cache
    ```
 
-9. Start the local development server.
+9a. Start the local development server.
+   ```bash
+   $ yarn watch
+   ```
+
+   The watch is complete when the CLI says `Compiled successfully`.
+
+10a. Open [http://localhost:3001](http://localhost:3001) in a browser
```

Comment:
```suggestion
10a. Open [http://localhost:3001](http://localhost:3001) in a browser if you built `vets-website` using the scaffold option; otherwise, continue on to the [content-build](#content-build) section for viewing changes in the browser.
```
Since building `vets-website` without the scaffold option won't show the site in the browser, should we add something like this to this section, so it's explicit on how to see the website in the browser?